chart/flux/CHANGELOG.md

```diff
@@ -1,4 +1,13 @@
-## 0.9.1 (2019-04-29)
+## 0.9.3 (2019-05-08)
+
+### Improvements
+
+ - Updated Flux to `1.12.2` and Helm operator to `0.9.0`
+   [weaveworks/flux#2025](https://github.com/weaveworks/flux/pull/2025)
+ - Mount sub path of repositories secret
```

Comment:
Looking at the comparison of [`chart-0.9.2` vs `master`](https://github.com/weaveworks/flux/compare/chart-0.9.2...master), I think https://github.com/weaveworks/flux/pull/2004 is missing.